### PR TITLE
add sinc etc.

### PR DESCRIPTION
**Pull request by zpceng314**
add sinc etc.

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/tensor.Torch.count_nonzero.md [hunk 1]
```diff
@@ -1,20 +0,0 @@
-## [ 仅 Paddle 参数更多 ] torch.Tensor.count_nonzero
-
-### [torch.Tensor.count_nonzero](https://pytorch.org/docs/stable/generated/torch.Tensor.count_nonzero.html?highlight=count_non#torch.Tensor.count_nonzero)
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/tensor.Torch.count_nonzero.md:3]: 这个文档没有新的吗

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histc.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ torch.Tensor.histc(bins=100, min=0, max=0)
 paddle.Tensor.histogram(bins=100, min=0, max=0, name=None)
 ```
 
-两者功能一致且参数完全一致。
+两者功能一致，但返回参数类型不一致，Pytorch 返回数据类型与 Tensor 一致， Paddle 默认返回 int64 类型。
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histc.md]: 与哪个Tensor一致？

Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histc.md]: 写：返回Tensor的数据类型不一致，写具体些

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.sinc.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+### 转写示例
+
+```python
+# Pytorch 写法
+y = a.sinc()
+
+# Paddle 写法
+import numpy
+y = paddle.where(a==0, x=paddle.to_tensor([1], dtype=a.dtype), y=paddle.sin(numpy.pi*a)/(numpy.pi*a))
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.sinc.md:19]: 这个第二行的y没有被用到，是不是转写示例有点问题？

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.SyncBatchNorm.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+#### momentum：
+```python
+# PyTorch 写法
+m = torch.nn.SyncBatchNorm(24, momentum=0.2)
+
+# Paddle 写法
+m = paddle.nn.SyncBatchNorm(24, momentum=0.8)
+```
+
+#### momentum：
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.SyncBatchNorm.md]: 这个是dtype吧

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.sinc.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+### 转写示例
+
+```python
+# Pytorch 写法
+y = torch.sinc(a)
+
+# Paddle 写法
+import numpy
+y = paddle.where(a==0, x=paddle.to_tensor([1], dtype=a.dtype), y=paddle.sin(numpy.pi*a)/(numpy.pi*a))
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.sinc.md:19]: 第二行的y，没有被其他地方使用，这个第二行的作用是？

File: docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.random.seed.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+```
+
+将生成随机数的种子设置为非确定性随机数。返回一个用于播种 RNG 的 64 位数字。
+
+PaddlePaddle 目前无对应 API，可使用如下代码组合实现该 API。
+
+###  转写示例
+```python
+# Pytorch 写法
+torch.seed()
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.random.seed.md]: 这个公开API是torch.random.seed还是torch.seed，选公开的写就可以，如果都公开就都写

File: docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.special.sinc.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+### 转写示例
+
+```python
+# Pytorch 写法
+y = torch.special.sinc(a)
+
+# Paddle 写法
+import numpy
+y = paddle.where(a==0, x=paddle.to_tensor([1], dtype=a.dtype), y=paddle.sin(numpy.pi*a)/(numpy.pi*a))
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.special.sinc.md:19]: 第二行的功能是？

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/tensor.Torch.count_nonzero.md (hunk 1 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/tensor.Torch.count_nonzero.md:3]: 有的，可能commit没加进去

File: docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.sinc.md (hunk 5 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/ops/torch.sinc.md:19]: 这个是github的现实问题，第二个y是paddle.where的输入参数

File: docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.special.sinc.md (hunk 7 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.special.sinc.md:19]: 这个是github的显示问题，第二个y是paddle.where的输入参数

File: docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.random.seed.md (hunk 6 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/others/torch.random.seed.md]: done!

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.sinc.md (hunk 3 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.sinc.md:19]: 这个是github的现实问题，第二个y是paddle.where的输入参数

File: docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.SyncBatchNorm.md (hunk 4 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/nn/torch.nn.SyncBatchNorm.md]: done!

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histc.md (hunk 2 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histc.md]: done!

Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.histc.md]: 指torch调用算子的那个Tensor

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/tensor.Torch.count_nonzero.md (hunk 1 above)
Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/tensor.Torch.count_nonzero.md:3]: 检查了下，已经存在repo里了

File: docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.pinverse.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+                   rcond=1e-15,
+                   hermitian=False,
+                   name=None)
+```
+
+其中 Paddle 相比 Pytorch 支持更多参数，具体如下：
+### 参数映射
+| PyTorch       | PaddlePaddle | 备注                                                   |
+| ------------- | ------------ | ------------------------------------------------------ |
+| -        | x            | 输入的 Tensor，Pytorch 无此参数。                   |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.pinverse.md]: 类成员函数直接从第二个开始对比

Comment (zpceng314) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/Tensor/torch.Tensor.pinverse.md]: done!

**Review by zhwesky2010 — APPROVED** [review]
